testsuite/testsuite.go

```diff
@@ -86,7 +86,7 @@ func FeatureContext(s *godog.Suite) {
 		ShouldBeInValidFormat)
 
 	// Command output and execution: extra steps
-	s.Step(`^with up to "(\d*)" retries with wait period of "(\d*(?:ms|s|m))" command "(.*)" output (?:should contain|contains) "(.*)"$`,
+	s.Step(`^with up to "(\d*)" retries with wait period of "(\d*(?:ms|s|m))" command "(.*)" output (should contain|contains|should not contain|does not contain) "(.*)"$`,
```

Comment:
@gbraad this line only matches the four options which are unambiguous regarding `not` in them.